hibernate-testing/src/main/java/org/hibernate/testing/cleaner/PostgreSQLDatabaseCleaner.java

```diff
@@ -52,7 +52,7 @@ public void clearAllSchemas(Connection connection) {
 				statement -> {
 					try {
 						return statement.executeQuery(
-								"SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME <> 'information_schema' AND SCHEMA_NAME <> 'sys' AND SCHEMA_NAME NOT LIKE 'pg_%'" );
+								"SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME <> 'information_schema' AND SCHEMA_NAME <> 'sys' AND SCHEMA_NAME <> 'public' AND SCHEMA_NAME NOT LIKE 'pg_%'" );
```

Comment:
I'm not really sure about this one, but when I created a database and user I didn't have permissions to trunc the `public` schema.

Reply:
@beikov thoughts? what permissions are needed to drop the `public` schema?

Reply:
Since our testing is usually done with some sort of superuser this didn't come up yet, but I think it is fine to ignore the `public` schema.

Reply:
Ah, right, OK, that explains it.